catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/GeoEntryQueryable.java

```diff
@@ -35,6 +35,17 @@
    */
   List<GeoEntry> query(String queryString, int maxResults) throws GeoEntryQueryException;
 
+  /**
+   * Retrieves the {@code GeoEntry} corresponding to the supplied identifier. {@code Suggestion} ids
+   * may be passed to this method to retrieve the GeoEntry for a specific suggestion.
+   *
+   * @param id {@code String} identifier used to retrieve a specific {@code GeoEntry}
+   * @return {@code GeoEntry} matching the identifier supplied or null if not found
+   * @throws IllegalArgumentException if {@code id} is null or empty
```

Comment:
❗️ remove or add exception to throws declaration

Reply:
IllegalArgumentException is a runtime exception

Reply:
good point 👍